src/qt/bitcoinunits.h

```diff
@@ -107,8 +106,11 @@ class BitcoinUnits: public QAbstractListModel
     static CAmount maxMoney();
 
 private:
-    QList<BitcoinUnits::Unit> unitlist;
+    QList<Unit> unitlist;
 };
 typedef BitcoinUnits::Unit BitcoinUnit;
 
+QDataStream& operator<<(QDataStream& out, const BitcoinUnit& unit);
+QDataStream& operator>>(QDataStream& in, BitcoinUnit& unit);
```

Comment:
75832fdc37ea3fe9cf515bd1946e220fe07a440b
```suggestion
QDataStream& operator>>(const QDataStream& in, BitcoinUnit& unit);
```

Reply:
I think it is ok as is. See https://doc.qt.io/qt-5.15/qdatastream.html#reading-and-writing-other-qt-classes.

Reply:
Sure but there's no reason for an in-param to be passed by reference instead of by reference to const, see https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const.